<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="paralelasButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Paralelas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="esfericoButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Esferico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cilindricoButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Cilindrico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacitores</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
